eng/testing/FindWasmHostExecutable.cmd

```diff
@@ -26,12 +26,12 @@ if exist "%__WasmBinaryPathWithoutExtension%.js" (
 )
 
 if "%__WasmBinaryExtension%" == ".wasm" (
-  if "%WASMER_EXECUTABLE%" == "" (
-    :: When running tests locally, assume wasmer is in PATH.
-    set WASMER_EXECUTABLE=wasmer
+  if "%WASMTIME_EXECUTABLE%" == "" (
+    :: When running tests locally, assume wasmtime is in PATH.
+    set WASMTIME_EXECUTABLE=wasmtime
   )
 
-  set WASM_HOST_EXECUTABLE="!WASMER_EXECUTABLE!" --
+  set WASM_HOST_EXECUTABLE="!WASMTIME_EXECUTABLE!" run -S http
```

Comment:
What does 'bare' `wasmtime run <component>` do (if anything)?

Reply:
It will run the component with host support for the `wasi:cli/run` world (which includes access to e.g. environment variables, filesystem, clocks, etc.).  The `-S http` option says to add host support for the `wasi:http/proxy` world as well, which is not part of `wasi:cli/run`.

BTW, `wasmtime serve <component>` provides host support for `wasi:http` but _not_ `wasi:cli` by default, but you can pass `-S cli` to add it.

Reply:
Interesting that one can combine worlds like that.